libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -354,11 +385,59 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
 
     /** @hidden */
     private _handleStepOrStatusChanges(): void {
-        this._setContentTemplates();
-        this._shrinkWhileAnyStepIsTooNarrow();
-        this._cdRef.detectChanges();
-        this._setContainerAndTallContentHeight();
-        this._scrollToCurrentStep();
+        if (this._isCurrentStepSummary()) {
+            this.progressBar.setCssDisplay(false);
+            this._showSummary();
+        } else {
+            this.progressBar.setCssDisplay(true);
+            this._setContentTemplates();
+            this._shrinkWhileAnyStepIsTooNarrow();
+        }
+        setTimeout(() => {
+            this._cdRef.detectChanges();
+            this.resizeHandler();
+            if (!this._isCurrentStepSummary()) {
+                this._scrollToCurrentStep();
+            }
+        });
+        this._setFinalStep();
+    }
+
+    /** @hidden */
+    private _setFinalStep(): void {
+        if (this.steps.last.isSummary && this.steps.toArray()[this.steps.length - 2]) {
+            this.steps.toArray()[this.steps.length - 2].content.tallContent = true;
+            this.steps.toArray()[this.steps.length - 2].finalStep = true;
+            // TODO: remove the line below when https://github.com/SAP/fundamental-styles/issues/1978 is addressed
+            this.steps.toArray()[this.steps.length - 2].completed = false;
+            this.steps.last.removeFromDom();
+        } else if (this.steps.last) {
+            if (this.steps.last.content) {
+                this.steps.last.content.tallContent = true;
+            }
+            this.steps.last.finalStep = true;
+        }
+    }
+
+    /** @hidden */
+    private _showSummary(): void {
+        const summary = this.steps.filter((step) => {
+            return step.isSummary;
+        });
+        summary[0].content.tallContent = true;
+        this.contentTemplates = [summary[0].content.contentTemplate];
```

Comment:
I think you can feel free to use find() there instead o filter, It will return first found item. Also remove unnecesary `{}`